tests/toolchain/gcc_compilation.pm

```diff
@@ -0,0 +1,35 @@
+# SUSE's openQA tests
+#
+# Copyright © 2017 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: C and C++ source compilation to check that gcc from toolchain module
+# Maintainer: Anton Smorodskyi <asmorodskyi@suse.com> , Michal Nowak <mnowak@suse.com>
```

Comment:
@mnowaksuse  if you don't mind  you would like to leave it like this :) as you still have a lot of expertise in this area